WTG.Analyzers.Utils.Test/ExpressionSyntaxFactoryTest.cs

```diff
@@ -48,7 +48,15 @@ public string LogicalNot(string expressionString)
 		[TestCase(1, ExpectedResult = "1")]
 		[TestCase(-1, ExpectedResult = "-1")]
 		[TestCase(65536, ExpectedResult = "65536")]
-		public string CreateLiteral(int value)
+		public string CreateLiteralInt(int value)
```

Comment:
Does this really need to be two different test methods?

Reply:
They test two different overloads. (`CreateLiteral(int)` and `CreateLiteral(string)`)